### PR TITLE
Copied accelerator accounting to InterTwin

### PR DESCRIPTION
**Pull request by Will-Cross1**
Resolves GT-538

Copied accelerator_accounting.json file from dashboards/iris to the dashboards/intertwin directory

Changed iris_accelerator dataset name to intertwin_accelerator inside the intertwin/accelerator_accounting.json file as is just an internal reference

Added the new file path to the providers inside the intertwin.yml file

### CODE REVIEW

**Comment by gregcorbett** [comment]
The change itself looks good to me. I have a couple of nitpicks about the commit message:

- 'Copyed' -> 'Copied'
- 'Copyed accelerator_accounting.json to the InterTwin directory': copied from where? It looks like the file just appears in the Git repo?
- 'Changed iris_accelerator to intertwin_accelerator': It's not clear to me what this refer to?

File: provisioning/dashboards/intertwin.yml [hunk 1]
```diff
@@ -9,3 +9,11 @@ providers:
   updateIntervalSeconds: 60 #how often Grafana will scan for changed dashboards
   options:
     path: /var/lib/grafana/dashboards/intertwin/intertwin.json
+- name: InterTwin Accelerator Dashboard
```
Comment (tofu-rocketry) [inline provisioning/dashboards/intertwin.yml:12]: Could you fix the capitalisation in this and line 4 to use "interTwin"?

File: dashboards/intertwin/accelerator_accounting.json [hunk 2]
```diff
@@ -0,0 +1,5317 @@
...
+    "from": "2019-07-06T13:46:59.039Z",
+    "to": "2025-02-12T13:46:59.039Z"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Accelerator Dashboard",
+  "uid": "h4vSOaw7z",
+  "version": 22,
+  "weekStart": ""
+}
```
Comment (tofu-rocketry) [inline dashboards/intertwin/accelerator_accounting.json:5317]: Add a newline at the end so that it follows the Unix standard.

**Review by tofu-rocketry — CHANGES_REQUESTED** [review]

File: provisioning/dashboards/intertwin.yml (hunk 1 above)
Comment (Will-Cross1) [reply provisioning/dashboards/intertwin.yml:12]: Should I change the other provider inside also?

**Review by gregcorbett — APPROVED** [review]
LGTM, you might want to squash the two commits once Adrian has reviewed as well.

File: dashboards/intertwin/accelerator_accounting.json (hunk 2 above)
Comment (Will-Cross1) [reply dashboards/intertwin/accelerator_accounting.json]: resolved in latest commit

**Review by tofu-rocketry — APPROVED** [review]

**Pull request merged**